tests/typescript/conformance/interfaces/interfaceDeclarations/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -44,8 +51,17 @@ interface Derived extends Base, Base2 {
   x: { b: string };
 }
 
-interface Derived2 extends Base, Base2 { // error
-  x: { a: number, b: string };
+// comment above TSInterfaceDeclaration
+interface Derived2 extends Base, Base2 // comment above TSInterfaceDeclaration
+{
+  // comment next TSInterfaceDeclaration
+  // comment above TSTypeLiteral
+  // comment inside of ObjectTypeAnnotation
+  // comment below TSTypeLiteral
```

Comment:
Prettier implements its own logic to place comments, it is not eslint-typescript related. What may trigger this is the location of nodes being incorrect.